src/components/formSections/components/TermsListField.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
+import { IntlConsumer } from '@folio/stripes/core';
 import {
   Button,
+  Card,
   Col,
   IconButton,
-  Label,
   Row,
   Select,
   TextArea,
   TextField,
 } from '@folio/stripes/components';
 
+import EditCard from '@folio/stripes-erm-components/lib/EditCard';
```

Comment:
Don't assume the directory structure of Stripes libraries. Instead, import its named exports. Eg, 
```suggestion
import { EditCard } from '@folio/stripes-erm-components';
```